### DISCUSSION TITLE
Specific Rule Handling

### DISCUSSION THREAD

**Discussion by mohtamayank1512** [oxsecurity/megalinter#4088] [Q&A]
Hello, I am trying to handle specific rules while using ARM_ARM_TTK linter but unable to disable the rule or change it from error to warning. I have updated the config file as below

linters:
ARM_ARM_TTK:
rules:
- id: 'Variables-Must-Be-Referenced'
level: 'warning'

Can you please help with the right kind of syntax? Also how do we find the correct rule ids.

If we want to add custom rules, can you share some guidance on this as well.

**Comment by nvuillam** [comment]
@mohtamayank1512 if you find such syntax in ChatGpt... sorry but it's an hallucination ^^

From what I see in the documentation, you can create a file **.arm-ttk.psd1** at the root of the repository

You can use [default MegaLinter one](https://github.com/oxsecurity/megalinter/blob/main/TEMPLATES/.arm-ttk.psd1) as template then customize it :)

**Reply by mohtamayank1512** [reply]
Hello Nicolas @nvuillam ,

Appreciate for your response. Yes I have used .arm-ttk.psd1 to include specific rules. I used ChatGPT to find a way if I can supress the error if a rules fails to validate and report is as a warning similar to BICEP Linter where we can mention if a rule should be reported as warning or error or off. I couldn't find any documentation regarding that on the Megalinter site or any other GitHub repo.

If you have any thoughts kindly share.

/Mayank

**Reply by nvuillam** [reply]
@mohtamayank1512 this is how the powershell command is built -> https://github.com/oxsecurity/megalinter/blob/deee18b30b591cb174231adaf64c0e96b0899065/megalinter/linters/ArmLinter.py#L15

You can see it in logs if you define `LOG_LEVEL: DEBUG` in your .mega-linter.yml file

I'm sorry I never used arm-ttk, so I won't be a big help :( 

If you paste here the exact command line called by MegaLinter in your project, maybe experts like @EtienneDeneuve will be able to help :)
